config/config.go

```diff
@@ -50,6 +50,10 @@ var (
 			Value: defaultQueueType,
 			Usage: `The name of the queue type to use ("amqp", "http", or "file")`,
 		}),
+		NewConfigDef("AmqpHeartbeat", &cli.DurationFlag{
+			Value: 10 * time.Second,
+			Usage: "The heartbeat timeout value defines after what time the peer TCP connection should be considered unreachable",
```

Comment:
This is a bit confusing, since the heartbeat is an AMQP-level concept and is only indirectly related to TCP. AFAIK triggering periodic traffic on an idle connection imitates TCP keepalive and allows clients to detect a broken connection.

Reply:
This is the definition from the docs: https://www.rabbitmq.com/heartbeats.html#heartbeats-timeout. 
Do you have any suggestions on how to make it less confusing? 😅

Reply:
Fair enough! :)